yarn-project/aztec-node/src/aztec-node/server.ts

```diff
@@ -285,8 +287,27 @@ export class AztecNodeService implements AztecNode {
     await this.p2pClient!.sendTx(tx);
   }
 
-  public getTx(txHash: TxHash): Promise<L2Tx | undefined> {
-    return this.blockSource.getL2Tx(txHash);
+  public async getTxReceipt(txHash: TxHash): Promise<TxReceipt> {
```

Comment:
Moved here from PXE because I needed an access to block source now because TxEffect does not contain blockhash (unlike `L2Tx`).